components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -0,0 +1,57 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:AutomaticPointsTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Json = require('Module:Json')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local AutomaticPointsTable = Class.new(
+	function(self, frame)
+		self.frame = frame
+		self.args = Arguments.getArgs(frame)
+	end
+)
+
+function AutomaticPointsTable.run(frame)
+	local pointsTable = AutomaticPointsTable(frame)
+
+	pointsTable:extractPositionBackgroundData()
+
+	pointsTable:extractTournaments()

```

Comment:
You're not extracting tournaments anywhere, you're parsing them. see above.

Reply:
Noted